air/src/air.rs

```diff
@@ -94,16 +94,39 @@ pub trait AirBuilder: Sized {
         let x = x.into();
         self.assert_zero(x.clone() * (x - Self::Expr::ONE));
     }
+
+    fn assert_eq_ext<EF: AbstractExtensionField<Self::Expr>>(&mut self, x: EF, y: EF) {
+        let xb = x.as_base_slice();
+        let yb = y.as_base_slice();
+        for i in 0..EF::D {
+            self.assert_eq(xb[i].clone(), yb[i].clone());
+        }
+    }
+
+    fn assert_zero_ext<EF: AbstractExtensionField<Self::Expr>>(&mut self, x: EF) {
+        for xb in x.as_base_slice().iter().cloned() {
+            self.assert_zero(xb);
+        }
+    }
+
+    fn assert_one_ext<EF: AbstractExtensionField<Self::Expr>>(&mut self, x: EF) {
```

Comment:
I think this would usually be called the "all one" polynomial, while the "one" polynomial would usually have a single 1 coefficient. If we don't use either yet, maybe just remove for now?

Reply:
Removing for now.